larray/core/array.py

```diff
@@ -5923,8 +5969,14 @@ def to_excel(self, filepath=None, sheet_name=None, position='A1', overwrite_file
         header : bool, optional
             Whether or not to write a header (axes names and labels). Defaults to True.
         transpose : bool, optional
-            Whether or not to transpose the resulting array. This can be used, for example, for writing one dimensional
-            arrays vertically. Defaults to False.
+            Whether or not to transpose the resulting array. Defaults to False.
```

Comment:
This probably needs more explanation and/or an example. Maybe also saying that for 2D, it is equivalent to arr.transpose().to_excel()